### ISSUE TITLE
'error: In the GNU C Library, "major" is defined  by <sys/sysmacros.h>'

### ISSUE THREAD

**Issue by SimonSapin** [mozilla/rr#2003]
rr fails to build for me with GCC 6.3.1 and glibc 2.25 with errors like this:

```c
/tmp/yaourt-tmp-simon/aur-rr/src/rr-4.5.0/src/test/quotactl.c: In function ‘find_home_device’:
/tmp/yaourt-tmp-simon/aur-rr/src/rr-4.5.0/src/test/quotactl.c:31:13: error: In the GNU C Library, "major" is defined
 by <sys/sysmacros.h>. For historical compatibility, it is
 currently defined by <sys/types.h> as well, but we plan to
 remove this soon. To use "major", include <sys/sysmacros.h>
 directly. If you did not intend to use a system-defined macro
 "major", you should undefine it after including <sys/types.h>. [-Werror]
     if (maj == (int)major(home_stat.st_dev) &&
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~    
```

**Comment by SimonSapin** [comment]
rr fails to build for me with GCC 6.3.1 and glibc 2.25 with errors like this:

```c
/tmp/yaourt-tmp-simon/aur-rr/src/rr-4.5.0/src/test/quotactl.c: In function ‘find_home_device’:
/tmp/yaourt-tmp-simon/aur-rr/src/rr-4.5.0/src/test/quotactl.c:31:13: error: In the GNU C Library, "major" is defined
 by <sys/sysmacros.h>. For historical compatibility, it is
 currently defined by <sys/types.h> as well, but we plan to
 remove this soon. To use "major", include <sys/sysmacros.h>
 directly. If you did not intend to use a system-defined macro
 "major", you should undefine it after including <sys/types.h>. [-Werror]
     if (maj == (int)major(home_stat.st_dev) &&
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~    
```

**Comment by rocallahan** [comment]
This was fixed in revision 5a16d15ef348c069b82449dcdeaeea3c1eb8639b. Try updating to master.
